examples/digital-asset-manager/src/components/Categories.tsx

```diff
@@ -16,24 +16,22 @@ const Categories = () => {
     CategoryData[]
   >([]);
 
-  const listBynderCollections = () => {
-    fetch(`/.netlify/functions/authorize`).then(() => {
-      fetch(`/.netlify/functions/collections`)
-        .then((response) => {
-          return response.json();
-        })
-        .then((result) => {
-          setBynderCollectionNames([...result.collections]);
-          setLoading(false);
-        })
-        .catch((err) => {
-          console.error(err);
-        });
-    });
+  const listBynderCollections = async () => {
+    try {
+      await fetch(`/.netlify/functions/authorize`);
```

Comment:
You're not storing the result of this, dont you need to check if the authorization went well?

Reply:
For this call no, this authorize call will only be used to refresh the token pair - it will refresh the tokens if they are expired before calling the collections endpoint